birdy/native/converters.py

```diff
@@ -113,7 +118,7 @@ def convert_data(self, data):
             return netCDF4.Dataset(self.output.reference)
         except IOError:
             # download the file
-            temp_file, _ = urllib.urlretrieve(self.output.reference)
+            temp_file, _ = urlretrieve(self.output.reference)
```

Comment:
@huard, I didn't want to add `requests` as a dependency just for this, but I didn't think that owslib requires it... Maybe it would be better to use it here to handle errors better. Just a thought.

Reply:
I don't know enough about this to have an opinion. Please go ahead and do what you think is best.

Reply:
`requests` is a requirement of `OWSLib` ... so we can use it.